addons/skin.estuary/xml/DialogPVRChannelsOSD.xml

```diff
@@ -95,7 +95,7 @@
 							<width>80</width>
 							<height>70</height>
 							<aspectratio align="right">keep</aspectratio>
-							<texture>$INFO[listitem.icon]</texture>
+							<texture fallback="$INFO[Listitem.Icon]">$INFO[Listitem.EpgEventIcon]</texture>
```

Comment:
Makes no sense, see my other comment.
Please revert all other changes of this kind you made.

Reply:
In this screen the Icon fallback is not active, we have to have it.